From: "Rafael J. Wysocki" <rafael.j.wysocki@intel.com>
Date: Wed, 23 Sep 2020 13:50:12 +0200
Subject: ACPI: processor: Fix build for ARCH_APICTIMER_STOPS_ON_C3 unset
Patch-mainline: v5.9-rc7
Git-commit: 955921289dcdbc49b46a731ec07978fb7d4e1fc7
References: jsc#SLE-16407

Fix the lapic_timer_needs_broadcast() stub for
ARCH_APICTIMER_STOPS_ON_C3 unset to actually return
a value.

Fixes: aa6b43d57f99 ("ACPI: processor: Use CPUIDLE_FLAG_TIMER_STOP")
Reported-by: Guenter Roeck <linux@roeck-us.net>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 drivers/acpi/processor_idle.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/acpi/processor_idle.c
+++ b/drivers/acpi/processor_idle.c
@@ -176,6 +176,7 @@ static void lapic_timer_propagate_broadc
 static bool lapic_timer_needs_broadcast(struct acpi_processor *pr,
 					struct acpi_processor_cx *cx)
 {
+	return false;
 }
 
 #endif
